cypress/cypress.d.ts

```diff
@@ -20,7 +20,11 @@ declare global {
         database?: Partial<Database>;
         currentMember?: Member;
         appContext?: Partial<LocalContext>;
-        errors?: object;
+        errors?:
+          | {
+              deleteAppDataShouldThrow?: boolean | undefined;
+            }
+          | undefined;
```

Comment:
You dont need to specify undefined, the object is already marked as `?` which is the shorthand for `undefined`.

Reply:
Thanks for paying attention to that!